api/src/test/java/jakarta/data/OrderTest.java

```diff
@@ -45,9 +45,7 @@ void shouldCreateOrderFromList() {
         var list = List.of(Sort.desc("priority"));
         var order = Order.by(list);
 
-        SoftAssertions.assertSoftly(soft -> {
-            soft.assertThat(order.sorts()).containsExactlyElementsOf(list);
-        });
+        SoftAssertions.assertSoftly(soft -> soft.assertThat(order.sorts()).containsExactlyElementsOf(list));
```

Comment:
This does not appear to do anything except increase the length of a line beyond the recently agreed upon line length limitation.
```suggestion
        SoftAssertions.assertSoftly(soft ->
            soft.assertThat(order.sorts()).containsExactlyElementsOf(list));
```
It looks like this was done in several other places.